### PR TITLE
Add find meeting query function

### PR DESCRIPTION
**Pull request by danieltre23**

### CODE REVIEW

File: walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java [hunk 1]
```diff
@@ -15,9 +15,39 @@
...
+    }
+    if (24*60 - meetingStart >= request.getDuration()) {
+      possibleTimeRanges.add(TimeRange.fromStartDuration(meetingStart, 24*60 - meetingStart));
+    }
+    return possibleTimeRanges;
+  }
+
+  public ArrayList<Event> getRelevantEvents(Collection<Event> events, Collection<String> attendees) {
+    ArrayList<Event> relevantEvents = new ArrayList<Event>();
+    for (Event event: events) {
```
Comment (wujingya) [inline walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java:44]: nit: add a space before the colon

**Review by wujingya — COMMENTED** [review]
This week is test driven development, the idea is to write unit tests first and then make them pass by implementing the features. Sometimes it's easier to implement the features first, but let's also include unit tests in the same PR, this way it's easier for reviewers to see what's being tested.

**Comment by danieltre23** [comment]
Okay, I tested it first with the ones already in FindMeetingQueryTest.java, but the next step in the walkthrough is make my own tests, so I'll add them in this PR.

**Review by wujingya — APPROVED** [review]
Ah ok, sorry didn't know that there were existing tests.

**Pull request merged**